packages/core/src/providers/zitadel.ts

```diff
@@ -111,13 +111,21 @@ export interface ZitadelProfile extends Record<string, any> {
  *
  * :::
  */
-export default function ZITADEL<P extends ZitadelProfile>(
+export default function Zitadel<P extends ZitadelProfile>(
   options: OAuthUserConfig<P>
 ): OIDCConfig<P> {
   return {
     id: "zitadel",
     name: "ZITADEL",
     type: "oidc",
+    profile(profile: P) {
```

Comment:
Is this require at all? Is by default doing this exact mapping?

Reply:
It's not required, you can take a look at the other OIDC providers like Auth0 for reference 🙌